plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -481,11 +482,13 @@ public static function register_post_types() {
 					'description'         => __( 'This is where store orders are stored.', 'woocommerce' ),
 					'public'              => false,
 					'show_ui'             => true,
+					'menu_icon'           => 'dashicons-text-page',
+					'menu_position'       => 56,
 					'capability_type'     => 'shop_order',
 					'map_meta_cap'        => true,
 					'publicly_queryable'  => false,
 					'exclude_from_search' => true,
-					'show_in_menu'        => current_user_can( 'edit_others_shop_orders' ) ? 'woocommerce' : true,
+					'show_in_menu'        => true,
```

Comment:
why this change?

Reply:
This moves the "Orders" menu item to top-level when HPOS is disabled (i.e. when it's using the post type)